data/jegrami.json

```diff
@@ -0,0 +1,33 @@
+ {
+  "name": "Jeremiah Igrami",
+  "bio": "YOUR__BIO",
```

Comment:
Please add your bio/about/headline here instead in place of `YOUR__BIO` placeholder.